cupboard_app/tests.py

```diff
@@ -288,6 +293,136 @@ def test_create_list_ingredient(self):
         self.assertEqual(ing3, None)
         self.assertEqual(ing4, None)
 
+    def test_get_user_lists_ingredients(self):
+        """
+        Testing test_get_user_lists_ingredients returns all lists from a user
+        """
+        create_user("test_user", "user@test.com")
+        test_user = get_user("test_user")
+
+        create_ingredient("test_ing", "test_type1")
+        create_measurement("test_unit")
+        ing1 = create_list_ingredient("test_ing", 500, "test_unit")
+        create_list("test_listname")
+        create_user_list_ingredients("test_user", "test_listname", ing1)
+        result = get_user_lists_ingredients(test_user.username, test_user.id)
+        user_lists = []
+        for i in result:
+            user_lists.append(i)
+
+        result_lists = []
+        result_lists.append(UserListIngredients.objects.get(user=test_user.id))
+
+        self.assertEqual(user_lists, result_lists)
+
+    def test_update_list_ingredient(self):
```

Comment:
Add test for updating the list if the user list was created without ingredients in it (which is the default). i.e.
```
create_user_list_ingredients("test_user", "test_listname", None)
update_list_ingredient("test_user", "test_listname", "test_ing", 25, "test_unit")
```

Reply:
done 👍